test/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Tests/Configurations/DefaultOpenApiConfigurationOptionsTests.cs

```diff
@@ -68,17 +69,31 @@ public void Given_OpenApiDocTitle_When_Instantiated_Then_Property_Should_Return_
         }
 
         [DataTestMethod]
-        [DataRow(null, 0)]
-        [DataRow("", 0)]
-        [DataRow("https://localhost", 1)]
-        [DataRow("https://localhost,https://loremipsum", 2)]
-        public void Given_HostNames_When_Instantiated_Then_Property_Should_Return_Value(string hostnames, int expected)
+        [DataRow(0, null)]
+        [DataRow(0, "")]
+        [DataRow(1, "https://localhost")]
+        [DataRow(2, "https://localhost", "https://loremipsum")]
+        [DataRow(2, "https://localhost", " https://loremipsum")]
+        [DataRow(1, "https://localhost "," ")]
+        public void Given_HostNames_When_Instantiated_Then_Property_Should_Return_Value(int expected, params string[] hostnames)
         {
-            Environment.SetEnvironmentVariable("OpenApi__HostNames", hostnames);
+            Environment.SetEnvironmentVariable("OpenApi__HostNames", string.Join(",", hostnames));
 
             var options = new DefaultOpenApiConfigurationOptions();
 
             options.Servers.Should().HaveCount(expected);
+
+            if (expected > 0)
+            {
+                var expectedHostnames = hostnames
+                    .Where(p => !string.IsNullOrWhiteSpace(p))
+                    .Select((s, i) => (i, s.Trim()));
+
+                foreach (var (index, hostname) in expectedHostnames)
+                {
+                    options.Servers[index].Url.Should().Be(hostname);
+                }
+            }
```

Comment:
Can we split this part into a separate test method so that the  new test method takes care of only that part?